dftimewolf/lib/collectors/bigquery.py

```diff
@@ -22,17 +22,14 @@ def __init__(self,
                critical: bool = False) -> None:
     """Initializes a GCP logs collector."""
     super(BigQueryCollector, self).__init__(state, name=name, critical=critical)
-    self._project_name = ""
-    self._query = ""
-    self._description = ""
-    self._pandas_output = False
+    self._project_name: str = ''
 
   # pylint: disable=arguments-differ
   def SetUp(self,
-            project_name: str,
-            query: str,
-            description: str,
-            pandas_output: bool) -> None:
+            project_name: str = '',
+            query: str = '',
+            description: str = '',
+            pandas_output: bool = False) -> None:
```

Comment:
Any reason we're using default arguments here? We removed them from all other modules, to prefer setting defaults in the recipe instead.

Reply:
Fixed.